public/styles/components/_chart.scss

```diff
@@ -3,6 +3,12 @@ $loadingLabelHeight: 300px;
 
 .chart-wrap {
   position: relative;
+  background: #393938;
+  border-top: 1px solid #ffbc01;
```

Comment:
This is one of the generic colours in the `_variables.scss` file. It would probably be better to refer to this if possible.